tox.ini

```diff
@@ -21,6 +21,12 @@ commands =
     flake8 .
     black --check .
 
+[testenv:mypy]
```

Comment:
this doesn't seem to be running in CI

Maybe it's not important seeing as `flake8` and `black` seem missing from CI as well

Reply:
It needs to be added to `.travis.yml` to run. Those are both run there.

Reply:
Oh gotcha, thanks!